# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  609e318d1601c985b5b0056ff2f68318f4f22855
Misc: Show folder path in message list header tooltip.

diff --git a/mail/base/content/about3Pane.js b/mail/base/content/about3Pane.js
--- a/mail/base/content/about3Pane.js
+++ b/mail/base/content/about3Pane.js
@@ -4819,9 +4819,9 @@ var threadPaneHeader = {
       return;
     }
 
     this.folderName.textContent = gFolder?.abbreviatedName ?? document.title;
-    this.folderName.title = gFolder?.prettyName ?? document.title;
+    this.folderName.title = gFolder?.prettyPath ?? document.title;
     this.updateMessageCount(
       gFolder?.getTotalMessages(false) || gDBView?.numMsgsInView || 0
     );
     this.updateSelectedCount();
